DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DSACampaignsServlet.java

```diff
@@ -42,10 +42,16 @@
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String correspondingKeywordCampaignId = request.getParameter("keywordCampaignId");
+        String correspondingQueryId = request.getParameter("keywordCampaignId");
+        String queryName = "keywordCampaignId";
+
+        if (request.getParameter("hangouts") != null) {
+          correspondingQueryId = request.getParameter("userId");
+          queryName = "userId";
```

Comment:
We'll talk about this during the pod sync/hash code meeting today, but we are only pulling DSA campaigns for the user that's logged in, so some of this code will have to be edited.

Reply:
Sounds good, I think I should be able to incorporate UserService.